client/app/scripts/superdesk-archive/views/list.html

```diff
@@ -2,7 +2,7 @@
 	<div class="button-stack right-stack">   
 	    <div class="navbtn dropdown strict highlights-dropdown" 
             dropdown sd-package-highlights-dropdown 
-            ng-if="type === 'archive' && selected.desk">
```

Comment:
@petrjasek  At line 21 is still used selected.desk and it looks like is not set correctly when current desk selection is changed. I was not able to fix it.